kustomize/overlays/local/istio/jwt-request-authentication.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
 spec:
   jwtRules:
     - issuer: "http://<REPLACE_WITH_MINIKUBE_IP>:30007/auth/realms/event-bridge-fm"
-      jwksUri: "http://keycloak.keycloak:8180/auth/realms/event-bridge-fm/protocol/openid-connect/certs"
+      jwksUri: "http://keycloak.keycloak:8180/auth/realms/event-bridge-fm/protocol/openid-connect/certs"
```

Comment:
is this change needed to be part of this PR?

Reply:
No that is not a part. will fix it